### ISSUE TITLE
Suggestion: Animated splash screen

### ISSUE THREAD

**Issue by laurasalisbury** [thunderbiscuit/padawan-wallet#386]
Would there be interest in developing a simple splash screen for the app? Here is an [example](https://lottiefiles.com/free-animation/splash-screenjson-fDUuSHxVKS) of what I am thinking.

**Comment by thunderbiscuit** [comment]
That'd be amazing! Is this something you'd have time/skill to work on? I can try to find resources on what the usual file types for these are and post here.

**Comment by thunderbiscuit** [comment]
I think an animated vector drawable would be best, but it also looks like you can use Lottie files (I don't really know anything about them though).

Some links for future reference:
1. https://www.youtube.com/watch?v=Oy0oXwv3kSc
2. https://www.youtube.com/watch?v=eFZmMSm1G1c

**Comment by laurasalisbury** [comment]
Yes I can work on this when I can. I will start with [this file](https://github.com/thunderbiscuit/padawan-wallet/blob/master/padawan.svg). Thanks for the videos
